.github/workflows/package.yml

```diff
@@ -0,0 +1,34 @@
+name: Package test
+
+on:
+  pull_request:
+  issue_comment:
+    types: [created]  # run when comment added with just "📦"
+
+jobs:
+  package:
+    if: ${{ github.event_name == 'pull_request' || (github.event_name == 'issue_comment' && github.event.issue.pull_request != null && github.event.comment.body == '📦') }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+          ref: ${{ format('refs/pull/{0}/merge', github.event.pull_request.number || github.event.issue.number) }}
+
+      - id: get_pr
+        if: ${{ github.event_name == 'issue_comment' }}
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const pr = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.payload.issue.number
+            });
+            core.setOutput('base_sha', pr.data.base.sha);
+            core.setOutput('head_sha', pr.data.head.sha);
+
+      - uses: packagecontrol/st-package-reviewer-action@de58167cfca93775fcbf5061cce6e452b51ce7b6
```

Comment:
Please add a comment here that this commit hash refers to a specific branch and is not accessible from the default branch (yet). I can totally see this becoming a point of confusion in the future.